Source/WebCore/page/linux/ResourceUsageThreadLinux.cpp

```diff
@@ -312,11 +313,18 @@ void ResourceUsageThread::platformCollectMemoryData(JSC::VM* vm, ResourceUsageDa
     data.categories[MemoryCategory::GCHeap].dirtySize = currentGCHeapCapacity;
     data.categories[MemoryCategory::GCOwned].dirtySize = currentGCOwnedExtra - currentGCOwnedExternal;
     data.categories[MemoryCategory::GCOwned].externalSize = currentGCOwnedExternal;
+
+    int imagesDecodedSize = 0;
+    callOnMainThreadAndWait([&imagesDecodedSize] {
+        imagesDecodedSize = MemoryCache::singleton().getStatistics().images.decodedSize;
+    });
+    data.categories[MemoryCategory::Images].dirtySize = imagesDecodedSize;
+
     size_t categoriesTotalSize = 0;
     for (auto& category : data.categories)
         categoriesTotalSize += category.totalSize();
-    data.categories[MemoryCategory::Other].dirtySize = data.totalDirtySize - categoriesTotalSize;
 
+    data.categories[MemoryCategory::Other].dirtySize = data.totalDirtySize - categoriesTotalSize;
```

Comment:
Ah, I see it's here again. What changed then?

Reply:
It stays as it was: "other" category is calculated base on currentProcessMemoryStatus(memoryStatus.resident - memoryStatus.shared) minus what is in other categories(JS, Images currently)